pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -14,6 +14,11 @@
     <ShouldCreateLayout>false</ShouldCreateLayout>
     <PropsFile>$(IntermediateOutputPath)$(MSBuildProjectName).props</PropsFile>
     <PlatformManifestFile>$(IntermediateOutputPath)$(MSBuildProjectName).PlatformManifest.txt</PlatformManifestFile>
+    <!-- During an official build when we can guarantee that all RID-specific dependencies have been built,
+         restore all of those dependencies and gather the prospective content of the RID-specific Core.App
+         packages.  This is needed so that we have a complete platform manifest in the shipping version of
+         the Microsoft.NETCore.App (RID-agnostic/identity package). -->
+    <IncludeAllRuntimePackagesInPlatformManifest Condition="'$(SIGNED_PACKAGES)' == 'true'">true</IncludeAllRuntimePackagesInPlatformManifest>
```

Comment:
Yes I would definitely prefer to stop depending on env-variables in our builds they cause use so much grief because they are hidden away and different across setups.